modules/mapping-utils/src/createConnectionResolvers.js

```diff
@@ -1,15 +1,16 @@
+// @flow
 import { get } from 'lodash';
 
 import resolveAggregations from './resolveAggregations';
 import resolveHits from './resolveHits';
 
-type TcreateConnectionResolversArgs = {
+type TCreateConnectionResolversArgs = {
```

Comment:
not that this is the most important line of code in the entire application, but I kind of prefer it the other way. it's like:

```
someFn
```

```
TsomeFn // type information
```

capitalizing the `s` seems weird to me

Reply:
hmmm, yeah I suppose. I can revert :) just feels weird to not have camel case haha

Reply:
reverted :)